source/support/encrypting-files.rst

```diff
@@ -0,0 +1,66 @@
+.. _minio-support-encryption:
+
+================
```

Comment:
Hm.

So this only applies to `mc support inspect` - this might be more useful as a behavior section under that reference page rather than a full-page (as this should really only come up when directed to by engineering).

I kind of feel the same way about the `decoding-metadata` section as being more useful under `mc support inspect`

Both of these tools are likely only useful w/ SUBNET, and even then only in specific circumstances. Giving them full pages feels a bit odd - but if we do want to do this, I would say make it a single page under Support as something like "Retrieving and Viewing Object Metadata"